packages/Webkul/API/Http/Controllers/Shop/SessionController.php

```diff
@@ -103,7 +103,7 @@ public function update()
             'password'      => 'confirmed|min:6',
         ]);
 
-        $data = request()->all();
+        $data = request()->only('first_name','last_name','gender','date_of_birth','email','password');
 
         if (! $data['date_of_birth']) {
```

Comment:
Can, you please just remove these three lines because validation portion is handling `date_of_birth`. So not needed anymore.